homeassistant/components/tuya/strings.sensor.json

```diff
@@ -16,6 +16,22 @@
       "mild": "Mild",
       "good": "Good",
       "severe": "Severe"
+    },
+    "tuya__work_state": {
+      "auto": "Auto",
+      "manual": "Manual",
+      "idle": "Idle"
+    },
+    "tuya__weather_delay": {
+      "cancel": "Cancel",
+      "24h": "24h",
+      "48h": "48h",
+      "72h": "72h"
+    },
+    "tuya__smart_weather": {
+      "sunny": "Sunny",
+      "cloudy": "Cloudy",
+      "rainy": "Rainy"
     }
```

Comment:
This translation format isn't used anymore. Please rebase the PR and consult the current codebase to see how it is handled now (see `strings.json`).

../Frenck

Reply:
@frenck How is this supposed to be handle now?